laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -604,7 +620,7 @@ private function insertOption()
             Option::insert(
                 ['date' => $date, 'resident' => $resident, 'schedule' => $split[0],
                 'attending' => $attending, 'option' => $choice, 'milestones'=>$_REQUEST['milestones1'],
-                'objectives'=>$_REQUEST['objectives1'], 'isValid'=>1]
+                'objectives'=>$_REQUEST['objectives1'], 'anesthesiologist_id'=>$pref_anest1, 'isValid'=>1] 
```

Comment:
this formatting is pretty hard to read. I prefer:

```php
[
  'key1' => $value1,
  'key2' => $value2,
  'key3' => $value3,
  // etc
]
```

with that said, i'm fine with leaving it for now so long as we run a php formatter on the entire project as a first step of code cleanup/maintenance.